gnosis/verification.k

```diff
@@ -120,4 +120,14 @@ module VERIFICATION
 
     rule #bytesSegment(_, _, 0, _) => .WordStack
 
+// Range for #symEcrec
+//========================================================================
+    rule 0 <=Int  #symEcrec(DATA)             => true
+
+    //that's because the result in concrete semantics is trimmed to Address range.
+    rule          #symEcrec(DATA) <Int pow160 => true
+
+    //case 0 is never wrapped into #symEcrec(), corresponds to #ecrecEmpty(DATA) == true
+    rule 0 =/=Int #symEcrec(DATA)             => true
```

Comment:
maybe we also need this type of thing for the normal address validity condition:
```
andBool MASTER_COPY_ID >Int 0 andBool (notBool MASTER_COPY_ID in #precompiledAccounts(BYZANTIUM))
```

Reply:
I would add this condition only if I get issues. So far it's not needed.